<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilesAutoForm</class>
 <widget class="QWidget" name="FilesAutoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="templateGenerateGrp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>671</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>TemplatesAutoGenerate</string>
   </property>
   <widget class="QPushButton" name="GenBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="BothRadio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BothFiles</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="CRadio">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>.c File</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HRadio">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>.h File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NewFileNameEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>19</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>FileName:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="destCDirBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>311</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Dest c Dir</string>
    </property>
    <widget class="QTextBrowser" name="destCpathBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="srcFiledirChoseBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>C Dir Chose</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="destHDirBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>311</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Dest h Dir</string>
    </property>
    <widget class="QTextBrowser" name="destHpathBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="HeadFiledirChoseBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>H Dir Chose</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="cTempBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>311</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>c Template</string>
    </property>
    <widget class="QTextBrowser" name="cTempPathBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cTempChoseBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>C Template</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="hTempBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>311</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>h Template</string>
    </property>
    <widget class="QTextBrowser" name="hTempPathBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hTempChoseBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>H Template</string>
     </property>
    </widget>
   </widget>
   <zorder>hTempBox</zorder>
   <zorder>cTempBox</zorder>
   <zorder>destHDirBox</zorder>
   <zorder>destCDirBox</zorder>
   <zorder>GenBtn</zorder>
   <zorder>BothRadio</zorder>
   <zorder>CRadio</zorder>
   <zorder>HRadio</zorder>
   <zorder>NewFileNameEdit</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
